libs/auth-api-lib/src/lib/delegations/delegations-incoming.service.ts

```diff
@@ -1,11 +1,15 @@
-import { BadRequestException, Injectable } from '@nestjs/common'
+import { BadRequestException, Inject, Injectable } from '@nestjs/common'
 import { InjectModel } from '@nestjs/sequelize'
 
 import { User } from '@island.is/auth-nest-tools'
+import { NationalRegistryClientService } from '@island.is/clients/national-registry-v2'
+import { LOGGER_PROVIDER } from '@island.is/logging'
```

Comment:
You can do
```suggestion
import { type Logger, LOGGER_PROVIDER } from '@island.is/logging'
```
so you don't need a specific `import type....` statement

Reply:
Fixed.